.github/workflows/code-review.yml

```diff
@@ -1,16 +1,23 @@
 # This workflow triggers a code review using the OpenAI GPT-3.5 Turbo model with a 16k context window
+name: Code Review
 
-on: [pull_request]
+on:
+  pull_request: # Filter out draft pull requests
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
 
 jobs:
   code-review:
     runs-on: ubuntu-latest
     permissions:
       contents: read
       pull-requests: write
+    if: ${{ github.actor != 'slack-oss-bot' }}
```

Comment:
I still think the bot should ideally be able to recognize if a given PR is automated or just a dependency update, but I'm ok with this for now